composer.json

```diff
@@ -28,7 +28,8 @@
   },
   "require-dev": {
     "composer/composer": "1.6.* || 2.0.*@dev",
-    "composer/semver": "^1 || ^3"
+    "composer/semver": "^1 || ^3",
+    "deployer/deployer": "^6.8"
```

Comment:
No need to required deployer here, or do we?

Reply:
It's added as a dev dependency for autocomplete support etc..